Hardware/Hardware/CardReader/CardReaderServiceError.swift

```diff
@@ -192,6 +192,19 @@ public enum UnderlyingError: Error {
     /// Catch-all error case. Indicates there is something wrong with the
     /// internal state of the CardReaderService.
     case internalServiceError
+
+    case locationIdMissing
```

Comment:
I'd read `UnderlyingError` as being more bound to Stripe, but really that's just the extension with Stripe in the name, so adding this makes sense and avoids some of the polymorphism gymnastics I was having to do. Thanks!

Reply:
I think this could do with also being called `incompleteStoreAddress`, since we want to surface that error to the merchant, and will need to include the `adminUrl` associated value as well.

Reply:
Absolutely! I am terrible at naming things